stock_picking_on_hold/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Payment Method - Hold Pickings (until payment)",
+    "version": "15.0.1.0.0",
+    "website": "https://github.com/OCA/sale-workflow",
+    "category": "Warehouse",
+    "summary": "This module allows to hold the picking until the invoice is paid",
+    "depends": [
+        "account_payment_mode",
+        "sale_stock_picking_blocking",
+        "website_sale",
```

Comment:
IMHO, this is a too restrictive dependency as people do not want necessarily install website to get this functionality.

That part of code should be externalized in another module.